admin_guide/overcommit.adoc

```diff
@@ -42,6 +42,59 @@ the specified resource as is available with the lowest local priority. In low
 resource situations, containers that specify no resource requests are given the
 lowest quality of service.
 
+[[tune-buffer-chunk-limit]]
+=== Tune Buffer Chunk Limit
+
+If Fluentd logger is unable to keep up with a high number of logs, you will need
+to increase the compute resource values. The memory limit is used to calculate
+the Fluentd `buffer_queue_limit` by dividing the resource memory limit by the
+number of output multiplied by the `buffer_chunk_size`. The default
```

Comment:
Nice work.  We are learning that the amount of memory used by fluentd is not just from the size of the queue, but also from the number of internal buffer chunk "objects" it keeps around for handling fluentd records tagged a certain way.  There is work to reduce this down one, but we'll need to review this carefully.

The user has the ability to set the `buffer_chunk_size` and the `buffer_queue_limit` for fluentd, and then also set the memory limits.  Do we think we should describe what happens when the user specifies all three parameters and a conflict occurs?